test/myxql_test.exs

```diff
@@ -189,7 +189,8 @@ defmodule MyXQLTest do
       {:ok, pid} = MyXQL.start_link(@opts ++ [prepare: :unnamed])
       {:ok, query} = MyXQL.prepare(pid, "1", "SELECT 1")
       {:ok, query2, _} = MyXQL.execute(pid, query, [])
-      assert query == query2
+      assert query.ref != query2.ref
+      assert query.statement_id != query2.statement_id
```

Comment:
Not 100% sure about this one

Reply:
We should be executing the prepared query without repreparing it.